### DISCUSSION TITLE
Tilt view

### DISCUSSION THREAD

**Discussion by litLizard69** [organicmaps/organicmaps#1550] [Ideas]
I think it would be a useful feature to tilt the view so we can see the buildings from a different perspective. This is already possible in navigation mode but not in normal map-view.

I hope you developers could implement this!

**Comment by biodranik** [comment]
We can do everything. It's just a question of time, resources and priorities. This feature is an interesting one.

**Comment by arnaudvergnet** [comment]
It could also be useful to control the perspective while in navigation. Tilted camera while moving is very good as it allows you to see far away, but sometimes you just want to move the camera and zoom in to a place far away from the gps, but the camera automatically tilts the more you zoom, which makes it harder to focus on the place you want.

I suggest enabling auto perspective only when the camera is set to follow the gps, otherwise let the user control it with 2 figers.
